Lib/cgitb.py

```diff
@@ -124,7 +124,7 @@ def html(einfo, context=5):
         args, varargs, varkw, locals = inspect.getargvalues(frame)
         call = ''
         if func != '?':
-            call = 'in ' + strong(func) + \
+            call = 'in ' + strong(func.replace("<", "&lt;").replace(">", "&gt;")) + \
```

Comment:
You could call _pydoc.html.escape_ like the rest of the code. Maybe it doesn’t matter too much here, but in general it is important to escape ampersands.

Reply:
@vadmium Thank you for the review. :) It's fixed now.